manager/integration/tests/test_rwx.py

```diff
@@ -41,40 +41,14 @@ def write_data_into_pod(pod_name_and_data_path):
 @pytest.fixture(scope="module", autouse="True")
 def nfs(request):
 
-    api = get_core_api_client()
     apps_api = get_apps_api_client()
 
     cmd = ["kubectl", "apply", "-f", LONGHORN_NFS_INSTALLATION_URL]
     subprocess.check_output(cmd)
 
-    worker_cnt = 0
-    node_list = api.list_node()
-    for node in node_list.items:
-        if "csi.volume.kubernetes.io/nodeid" in node.metadata.annotations:
-            worker_cnt = worker_cnt + 1
-
-    cmd = ["kubectl get pod -l app={} | "
-           "awk 'NR>1 {{print $1}}'".format(LONGHORN_NFS_DAEMONSET_NAME)]
-    for i in range(RETRY_COUNTS):
-        nfs_pods = {}
-        pod_list = \
-            subprocess.check_output(cmd, shell=True).decode("utf-8").split()
-
-        for pod_name in pod_list:
-            nfs_pods[pod_name] = \
-                api.read_namespaced_pod(name=pod_name,
-                                        namespace='default').status.phase
-
-        if len(nfs_pods) == worker_cnt:
-            if all(value == "Running" for value in nfs_pods.values()):
-                break
-
-        time.sleep(RETRY_INTERVAL)
-
-    if not \
-            all(value == "Running" for value in nfs_pods.values()) \
-            or not len(nfs_pods) == worker_cnt:
-        raise Exception("Longhorn-nfs not installed on all nodes")
+    cmd = ["kubectl", "rollout", "status",
+           "ds/{}".format(LONGHORN_NFS_DAEMONSET_NAME), "--timeout=5m"]
+    subprocess.check_output(cmd)
 
     def finalizer():
         apps_api.delete_namespaced_daemon_set(name=LONGHORN_NFS_DAEMONSET_NAME,
```

Comment:
nit: we could consider deleting the DaemonSet by `kubectl delete -f LONGHORN_NFS_INSTALLATION_URL`.